### PR TITLE
add dashboard usage guide for HTTPChaos; fix some wrong contents

### PR DESCRIPTION
**Pull request by Hexilee**
Signed-off-by: xixi <i@hexilee.me>

### CODE REVIEW

File: i18n/zh/docusaurus-plugin-content-docs/current/simulate-http-chaos-on-kubernetes.md [hunk 1]
```diff
@@ -26,9 +26,21 @@ HTTPChaos 支持多种类型故障的组合。在创建 HTTPChaos 实验时，
...
 
-## 创建实验
+## 使用 Dashboard 创建实验
 
-Chaos Mesh 目前仅支持使用 YAML 配置文件创建 HTTPChaos 实验。在 YAML 配置文件中，你可以模拟一种 HTTP 故障类型，也可以模拟多种 HTTP 故障的组合。
+1. 单击实验页面中的“新的实验”按钮创建实验：
+
+![创建实验](./img/create-new-exp.png)
+
+2. 在“选择目标”处选择 “HTTP 故障”，然后选择具体行为（如 `RESPONSE ABORT`），并填写具体配置：
```
Comment (en-jin19) [inline i18n/zh/docusaurus-plugin-content-docs/current/simulate-http-chaos-on-kubernetes.md:35]:
```suggestion
2. 在“选择目标”处选择“HTTP 故障”，然后选择具体行为（如 `RESPONSE ABORT`），并填写具体配置：
```

**Review by en-jin19 — APPROVED** [review]
Rest LGTM

**Review by STRRL — APPROVED** [review]
LGTM!

File: i18n/zh/docusaurus-plugin-content-docs/current/simulate-http-chaos-on-kubernetes.md (hunk 1 above)
Comment (Hexilee) [reply i18n/zh/docusaurus-plugin-content-docs/current/simulate-http-chaos-on-kubernetes.md]: fixed

**Pull request merged**